modules/installation-aws-regions-with-no-ami.adoc

```diff
@@ -25,3 +39,33 @@ A region without native support for an {op-system} AMI is not available to
 select from the terminal during cluster creation because it is not published.
 However, you can install to this region by configuring the custom AMI in the
 `install-config.yaml` file.
+endif::aws-china,aws-gov[]
+
+ifdef::aws-china,aws-gov[]
+ifdef::aws-china[Red Hat does not publish a Red Hat CoreOS (RHCOS) Amazon Machine Image (AMI) for the AWS China regions.]
+ifdef::aws-gov[Red Hat does not publish a Red Hat CoreOS (RHCOS) Amzaon Machine Image for the AWS government or secret regions.]
+
+Before you can install the cluster, you must:
+
+* Upload a custom RHCOS AMI.
+* Manually create the installation configuration file (`install-config.yaml`).
++
+You specify the AWS region, and the accompanying custom AMI, in the installation configuration file.
+
+You cannot use the {product-title} installation program to create the installation configuration file. The installer does not list an AWS region without native support for an {op-system} AMI.
```

Comment:
the installer just can not list AWS regions, but it can create install-config.yaml config file.
The normal process should be like:
1. create insatll-config.yaml
2. edit install-config.yaml, add rhcos AMI and update region.
3. create cluster

so I suggest update above contents to:
1. `Manually create the installation configuration file` -> `Manually edit the installation configuration file`
2. delete `You cannot use the {product-title} installation program to create the installation configuration file.`